nc-games/src/App.css

```diff
@@ -14,25 +15,51 @@
 }
 
 .App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+  background-color: #1C2235;
   display: flex;
   flex-direction: column;
   align-items: center;
   justify-content: center;
   font-size: calc(10px + 2vmin);
-  color: white;
+  color: #FBF9F9;
+  margin-bottom: 0;
 }
 
-.App-link {
-  color: #61dafb;
+@media only screen and (min-width: 800px) {
+  main {
+    width: 800px;
+    margin: auto;
+  }
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+nav {
+  padding-top: 0;
+  background-color: #73648E;
+  margin-top:0;
+  margin-bottom: 0;
+  height: 5vh;
+  line-height: 5vh;
+}
+
+.reviewList {
+  list-style-type: none;
+  padding-left: 0;
 }
+
+.reviewList li {
+  background-color: #FBF9F9;
+  border: 3px solid #73648E;
+  margin: 10px;
+}
+
+.reviewList-Header {
+  border: 2px solid #73648E;
+  margin: 5px;
+}
+
+.reviewList-Body {
+  border: 2px solid #73648E;
+  margin: 5px;
+  overflow:scroll;
```

Comment:
Its a small thing - but css naming you'll more commonly see with a double underscore - BEM - https://www.devbridge.com/articles/implementing-clean-css-bem-method/#:~:text=What%20is%20BEM%3F,by%20following%20some%20simple%20rules.